src/webfaf/blueprints/celery_tasks.py

```diff
@@ -298,7 +336,7 @@ def action_run(action_name):
     pt_form = PeriodicTaskForm(request.form)
     if request.method == "POST":
         if request.values.get("run") and action_form.validate():
-            run_action.delay(action_name, action_form.to_cmdline_dict())
+            run_action(action_name, action_form.to_cmdline_dict())
```

Comment:
What's the reason for running the task in-process instead of having a worker do it? This will likely lead to timeouts for long-running tasks.